src/Cogworks.CogFlare.Core/Composers/ExternalCachePurgeComposer.cs

```diff
@@ -6,10 +6,11 @@ public void Compose(IUmbracoBuilder builder)
     {
         var cogFlareSettings = builder.Config
             .GetSection(nameof(CogFlareSettings))
-            .Get<CogFlareSettings>();
+            .Get<CogFlareSettings>() ?? new CogFlareSettings();
 
         if (!cogFlareSettings?.IsEnabled ?? true)
```

Comment:
Since we have added a fallback for the null case in the line above we don't need this `?` operator in here 😉